### PR TITLE
Storage 2881

### PR DESCRIPTION
**Pull request by Tiggles**

### CODE REVIEW

**Comment by Tiggles** [comment]
Looks great 👍

File: frontend-web/webclient/app/UCloud/FilesApi.tsx [hunk 1]
```diff
@@ -123,19 +126,22 @@ const FileSensitivityVersion = "1.0.0";
 const FileSensitivityNamespace = "sensitivity";
 type SensitivityLevel = | "PRIVATE" | "SENSITIVE" | "CONFIDENTIAL";
 let sensitivityTemplateId = "";
-function findSensitivity(file: UFile): SensitivityLevel {
+function findSensitivityWithFallback(file: UFile): SensitivityLevel {
+    // TODO(Jonas): This assumes an inherit case would be 'Private', work needs to be done for backend.
+    // TODO(Jonas): The type is not correct if this needs to be cast for this to work
```
Comment (DanThrane) [inline frontend-web/webclient/app/UCloud/FilesApi.tsx:131]: Not sure I understand these comments. `PRIVATE` is the correct default.

File: frontend-web/webclient/app/UCloud/FilesApi.tsx [hunk 2]
```diff
@@ -660,6 +677,81 @@ class FilesApi extends ResourceApi<UFile, ProductStorage, UFileSpecification,
...
+    const onUpdate = React.useCallback(async (e: React.SyntheticEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+        try {
+            const value = selection.current?.value;
+            const reasonText = reason.current?.value ?? "No reason provided."
+            if (!value) return;
+            if (value === SensitivityLevelMap.INHERIT) {
+                // Find latest that is active and remove that one. At most one will be active.
+                const entryToDelete = file.status.metadata?.metadata[sensitivityTemplateId][0];
```
Comment (DanThrane) [inline frontend-web/webclient/app/UCloud/FilesApi.tsx:699]: The active one will have `.status.type` should be either `approved` or `not_required`. The current code just picks the latest entry, if I remember correctly.

**Review by DanThrane — CHANGES_REQUESTED** [review]

File: frontend-web/webclient/app/UCloud/FilesApi.tsx (hunk 1 above)
Comment (Tiggles) [reply frontend-web/webclient/app/UCloud/FilesApi.tsx:131]: I think the first one was in reference to have "INHERIT" as an explicit option in the backend.

The second line is in reference to a `as any`-cast that's was corrected.

**Review by DanThrane — APPROVED** [review]

**Pull request merged**